### ISSUE TITLE
config.py fills up with import statements on model switching

### ISSUE THREAD

**Issue by changeling** [continuedev/continue#425]
**Describe the bug**

`config.py` is filling with redundant import statements, adding a new one on each model switch.

**To Reproduce**

Choose a model from the bottom menu.
Examine `config.py`.
Choose another, then switch back to previous.
See `config.py`.

**Expected behavior**
No redundant imports.

**Environment**
- Operating System: MacOS
- Python Version: 3.11
- Continue Version: At least >=v0.0.344

**Additional context**
```
"""
This is the Continue configuration file.

See https://continue.dev/docs/customization to for documentation of the available options.
"""
from continuedev.src.continuedev.libs.llm.ggml import GGML
from continuedev.src.continuedev.libs.llm.ggml import GGML
from continuedev.src.continuedev.libs.llm.ggml import GGML
from continuedev.src.continuedev.libs.llm.ggml import GGML
from continuedev.src.continuedev.libs.llm.together import TogetherLLM
from continuedev.src.continuedev.libs.llm.replicate import ReplicateLLM
from continuedev.src.continuedev.libs.llm.ollama import Ollama
from continuedev.src.continuedev.libs.llm.ggml import GGML
from continuedev.src.continuedev.libs.llm.anthropic import AnthropicLLM
from continuedev.src.continuedev.libs.llm.maybe_proxy_openai import MaybeProxyOpenAI
from continuedev.src.continuedev.libs.llm.maybe_proxy_openai import MaybeProxyOpenAI
from continuedev.src.continuedev.libs.llm.maybe_proxy_openai import MaybeProxyOpenAI
from continuedev.src.continuedev.libs.llm.ggml import GGML
from continuedev.src.continuedev.libs.llm.llamacpp import LlamaCpp
...
```

**Comment by sestinj** [comment]
You're right, and I know exactly where this is happening. I'll fix tonight. Also have noticed that sometimes it doubles models in the unused array, I'll fix that as well

**Comment by sestinj** [comment]
First of these two is resolved in 0.0.351

**Comment by sestinj** [comment]
...and the second will be in a new version tomorrow morning. Closing the issue, thx for the report!
